sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -125,6 +125,29 @@ class DatalabAppElement extends Polymer.Element {
     }
   }
 
+  /**
+   * Shows a notification toast at the bottom of the page with the given message.
+   * @param message message to show in the notification toast
+   * @param sticky whether the toast should stick around until explicitly dismissed in code
+   */
+  showNotification(message: string, sticky?: boolean) {
```

Comment:
I'm planning to also use this notification for quick snackbar-style messages, such as when a file operation succeeds. In that case, a different message will be shown, and sticky will be set to false.